build/ci/create_demo_pr.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -xe
+set +o pipefail
+
+sed -i "s+$current_operator_image+$wanted_operator_image+g" $csv_file
+
+# install gh command
+curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+sudo apt update
+sudo apt install gh
```

Comment:
`WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`
consider using `apt-get`?

Reply:
done